src/sdk/object_sdk.js

```diff
@@ -227,6 +249,7 @@ class ObjectSDK {
             this.requesting_account = await account_cache.get_with_cache({
                 bucketspace: this._get_bucketspace(),
                 access_key: token ? token.access_key : anonymous_access_key,
+                validation_callback: _validate_account,
```

Comment:
how this works on containerized?

Reply:
The function is implemented in `object_sdk` and in all the cases when it is called it should not have stat the files, as it is not relevant, and only check the validation by time.